core-skills/frontend-web-development/README.md

```diff
@@ -0,0 +1,45 @@
+# Frontend web developement [HTML/CSS]
+
+## Aim
+This course is intended to guide learning to a level where you can write and contribute semanitic, and maintainable HTML and CSS to a project.
+
+The course guides you through a typical "first page" build of any green field project. This encompasses selecting/creating a responsive CSS grid, building core shared components for your new site, and finally delivering an accessible, SEO optimised, lightning fast loading product detail page.
+
+## Assessment Style
+This core skill assessment is split into two parts. Firstly, you'll take a design from PSD to HTML/CSS during Learn Tech. This is assessed over three levels; Rhino, Narwhal, and Unicorn. Secondly, before completing each level as HTML/CSS is quite subjective you'll be asked to talk about the decisions you made whilst building out the deliverables for each level.
```

Comment:
Not sure about PSD - we should probs only require from PNG or PDF.

Reply:
On a delivery, you'd typically get a PSD or a Sketch file. I feel that frontend engineers should be comfortable working with the format

Reply:
True - I’d argue that good web designers wouldn’t use PSDs but that’s a separate problem to what this PR is attempting to solve